source/memory/memory.go

```diff
@@ -49,5 +67,20 @@ func (s Source) Discover() (source.Features, error) {
 		features["numa"] = true
 	}
 
+	// Check transparent_hugepage
+	choice, err := GetChoice("/host-sys/kernel/mm/transparent_hugepage/enabled")
+	if err != nil {
+		return nil, fmt.Errorf("can't read /host-sys/kernel/mm/transparent_hugepage/enabled: %s", err.Error())
```

Comment:
Hmm, I think returning "nil, err" on error is not the best thing to do. Even if detection of THP fails, we could advertise numa status. So, I think you should change all these returns to just log.Printf("ERROR:...") – also the numa case above

Reply:
Done.